astropy/io/fits/tests/test_connect.py

```diff
@@ -74,6 +75,12 @@ def test_overwrite_with_path(self, tmp_path):
             t1.write(Path(filename), format="fits", overwrite=True)
         t1.write(Path(tmp_path / filename), format="fits", overwrite=True)
 
+    def test_write_to_fileobj(self):
+        # regression test for https://github.com/astropy/astropy/issues/17703
+        t = Table(self.data)
+        with TemporaryFile() as temp:
+            t.write(temp, format="fits")
```

Comment:
As it seems this was not tested, can you check the output to be sure something is actually written ?

Reply:
done !